autovalet_husky/rviz/config.rviz

```diff
@@ -6,9 +6,9 @@ Panels:
       Expanded:
```

Comment:
not pushing rviz configs anymore, right?

Reply:
removed